config/set/code-quality.php

```diff
@@ -183,4 +184,5 @@
     $services->set(FlipTypeControlToUseExclusiveTypeRector::class);
     $services->set(ExplicitMethodCallOverMagicGetSetRector::class);
     $services->set(DoWhileBreakFalseToIfElseRector::class);
+    $services->set(OptionalParametersAfterRequiredRector::class);
```

Comment:
The rule should not be registered explicitly here, as it might create huge amount of changes, undesired.
The use of this rule must remain explicit with aware registration.